package/MDAnalysis/coordinates/PDB.py

```diff
@@ -1064,6 +1068,57 @@ def write_next_timestep(self, ts=None, **kwargs):
         self._check_pdb_coordinates()
         self._write_timestep(ts, **kwargs)
 
+    @staticmethod
+    def _deduce_PDB_atom_name(atom):
+        """Deduce how the atom name should be aligned.
+
+        Atom name format can be deduced from the atom type, yet atom type is
+        not always available. This function uses the atom name and residue name
+        to deduce how the atom name should be formatted. The rules in use got
+        inferred from an analysis of the PDB. See gist at
+        <https://gist.github.com/jbarnoud/37a524330f29b5b7b096> for more
+        details.
+        """
+        ions = ('FE', 'AS', 'ZN', 'MG', 'MN', 'CO', 'BR',
+                'CU', 'TA', 'MO', 'AL', 'BE', 'SE', 'PT',
+                'EU', 'NI', 'IR', 'RH', 'AU', 'GD', 'RU')
+        # Mercurial can be confused for hydrogen gamma. Yet, mercurial is
+        # rather rare in the PDB. Here are all the residues that contain
+        # mercurial.
+        special_hg = ('CMH', 'EMC', 'MBO', 'MMC', 'HGB', 'BE7', 'PMB')
```

Comment:
@richardjgowers A conceptually simpler way would be to have these arrays at the class level. This way they get created only once, even if several PDB files are written. Would it work for you?